websites-html/aboutus.html

```diff
@@ -40,6 +40,33 @@ <h2 class="page-title">About Us</h2>
             Join us on our journey as we grow into the next generation of skilled frontend developers!</div>
     </div>
 
+<<<<<<< HEAD
```

Comment:
This is an unresolved merge conflict, you shouldn't push up code where theres merge conflicts that you haven't taken care off.

If you're not sure how to deal with merge conflicts then you should contact Rob